resources/generate_book.py

```diff
@@ -329,25 +329,56 @@ def make_book(rm: ResourceManager, i18n: I18n, local_instance: bool = False):
-            text('Different fruit plants will grow, flower, and fruit at different times of year.'),  # todo: finish
+            text('Fruit plants are seasonal. During their cold season, these plants will appear brown and lifeless. In the spring, they become green and healthy, getting ready to produce fruit and grow larger. The exact times this happen varies by the fruit, with no two being the same. Fruit plants can die, as well: of old age, and of improper climate conditions.'),
             page_break(),
-            text('', title='Fruit Trees').anchor('fruit_trees'),
-            image(),  # todo: image of fruit tree in the world, or a sophisticated multiblock
-            # todo: blurb on how to harvest and farm fruit trees in general
-            # todo: blurb on all fruit tree species
+            text('$(thing)Fruit trees$() grow from tiny saplings into large, flowering trees. The branches of fruit trees are their heart, and they will grow as long as the climate conditions are right. As fruit trees mature, they will grow $(thing)leaves$() all around their branches. The leaves can flower and fruit depending on the season.', title='Fruit Trees').anchor('fruit_trees'),
+            image('tfc:textures/gui/book/tutorial/fruit_tree.png', text_contents='A typical fruit tree.'),
+            text('Fruit trees start out at $(thing)Saplings$(). Saplings will only start growing, placing their first piece of the tree, if it is not the dormant season for that fruit. The size of the finished tree is loosely determined by how many saplings are in the original sapling block. More saplings means a bigger tree.$(br)More saplings can be added to a single block through $(thing)Splicing$(). To splice a sapling into another, just $(item)$(k:key.use)$() on it while holding a sapling and a $(thing)Knife$() in your off hand.'),
+            text('Saplings can also be placed on the first \'elbow\' sections a fruit tree produces, places where the branch is attached to one side and upwards. This allows one fruit tree to grow multiple fruits. Breaking these elbows with a $(thing)Axe$() also yields saplings. Harvesting a fruit tree leaf is done with $(item)$(k:key.use)$() when the fruit tree is at its fruiting stage. This will give one fruit, and revert the plant back to its growing stage, until it is time for it to become dormant.'),
             page_break(),
-            text('', title='Tall Bushes').anchor('tall_bushes'),
-            text(''),
-            # todo: blurb on how to harvest and farm tall bushes in general
-            # todo: blurb on all tall bush species
+            detail_fruit_tree_left('cherry'),  # todo: fancy tree vis?
+            detail_fruit_tree_right('cherry'),
```

Comment:
Rather than having these `left` and `right` methods, have one that returns a tuple. Then you can call it like

```python
    *detail_fruit_tree('cherry'),
    *detail_fruit_tree('green_apple')
```

And, you can also include explicit `page_break()`s:

```python
def detail_fruit_tree():
    return (
        page_break(),
        # the detail pages
```